trello/api/features/negative/ListNegative.feature

```diff
@@ -30,6 +27,43 @@ Feature: List
     And Sends request
     Then Should return status code 400
     And Validates response message with message "invalid value for id"
-    And Sets a "GET" request to "/lists/ListObject.id"
+
+
+  Scenario: Card with an incorrect id can not be found
+    When Sets a "GET" request to "/cards/5e3d75852b8afb5c7c60dc45invalidId/actions"
+    And Sends request
+    Then Should return status code 400
+    And Validates response message with message "invalid id"
```

Comment:
It is like this, this scenario is duplicated
Scenario: Can not get a card by incorrect idCard
    When Sets a "GET" request to "/cards/5e3d75852b8afb5c7c60dc45invalidId/actions"
    And Sends request
    Then Should return status code 400
    And Validates response message with message "invalid id"